src/collective/easyform/actions.py

```diff
@@ -412,7 +412,7 @@ def get_attachments(self, fields, request):
             xmlstr = output.getvalue()
             now = DateTime().ISO().replace(' ', '-').replace(':', '')
             filename = 'formdata_{0}.xml'.format(now)
-            attachments.append((filename, 'text/xml', 'utf-8', xmlstr))
+            attachments.append((filename, 'application/xml', 'utf-8', xmlstr))
```

Comment:
it's non-obvious why this results in base64 encoding so it would be helpful to the reader to comment why application mime is needed rather than text.